batcher/aligned-sdk/src/sdk.rs

```diff
@@ -99,6 +111,103 @@ pub async fn submit_multiple_and_wait_verification(
     Ok(aligned_verification_data)
 }
 
+/*
+Returns the estimated `max_fee` depending on the batch inclusion preference of the user, based on the max priority gas price.
+NOTE: The `max_fee` is computed from an rpc nodes max priority gas price.
+
+For batch cost estimates we estimate a batch will have ~32 proofs present.
```

Comment:
Batch size sounds like the kind of parameter you might want to tune eventually. It should probably be a global constant instead of a magic number, so we can change it in one place.